modules/webhook/variables.tf

```
@@ -162,3 +162,13 @@ variable "lambda_runtime" {
   type        = string
   default     = "nodejs14.x"
 }
+
+variable "lambda_architecture" {
+  description = "AWS Lambda architecture. Lambda functions using Graviton processors ('arm64') tend to have better price/performance than 'x86_64' functions. "
+  type        = string
+  default     = "arm64"
```

I prefer to keep the default the same as the current default architecture
```suggestion
  default     = "x86_64"
```